### PR TITLE
added color to pieces 

### PR DESCRIPTION
**Pull request by AliLynne**
White pieces now populate on create. Black pieces populate on update.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -46,7 +46,6 @@
     t.integer  "y2"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.string   "color"
```
Comment (Muzzh) [inline db/schema.rb]: @AliLynne 
I see a change in schema.rb, but no migration ?
won't this break somewhere down the line ?
I thought we weren't supposed to play with schema.rb unless from migrations

Comment (AliLynne) [reply db/schema.rb]: I didn't do that. It came from someone else's PR.

Comment (Muzzh) [reply db/schema.rb]: @AliLynne 
wait I'm confused. did Tim remove this in his PR ?

Comment (AliLynne) [reply db/schema.rb]: Looks like it. I was thinking it was removing the line from the chess_pieces not the moves. So I guess we're good?

**Comment by AliLynne** [comment]
I thought we'd decided not to populate the black pieces until the second player had joined. I think I can add a feature test to make sure the pieces actually show up.

**Comment by Muzzh** [comment]
@AliLynne 
yeah as I said it's not gonna be breaking anything.
I had thought we should change the association to optional and build an `assign_pieces(color)` method
which would make sure that we can verify on creation that all the pieces are there with the model validations. But, yeah, not an issue, just some thoughts I had about it hehe

**Comment by AliLynne** [comment]
@Muzzh added tests to check for game pieces. Right now it just checks to see if any of the pieces have loaded.

**Comment by AliLynne** [comment]
@Muzzh Can you double check to make sure I didn't muck anything up? If it's good, we can merge and hopefully fix the schema.rb issue as well as having a functioning create game with pieces.

**Pull request merged**